spec/sauce_helper.rb

```diff
@@ -8,6 +8,39 @@
 
 tunnel_id = "restricted-input"
 
+PLATFORM = ENV["PLATFORM"]
+
+def select_browsers
+  browsers = []
+
+  if !PLATFORM || PLATFORM == "desktop"
+    browsers += [
+      ["Windows 10", "chrome", nil],
+      # Firefox 48 (latest on Sauce) is failing all tests
```

Comment:
Minor: I think Firefox 49 is the latest on Sauce; perhaps we should update this comment.